packages/optimizer/lib/transformers/RewriteAmpUrls.js

```diff
@@ -109,13 +110,12 @@ class RewriteAmpUrls {
 
     // runtime-host and amp-geo-api meta tags should appear before the first script
     if (!this._usesAmpCacheUrl(host) && !params.lts) {
-      let versionlessHost;
-      if (this.isAbsoluteUrl_(params.ampUrlPrefix)) {
-        versionlessHost = calculateHost({ampUrlPrefix: params.ampUrlPrefix});
-      } else {
-        versionlessHost = '.';
+      try {
+        const url = new URL(host);
+        this._addMeta(head, 'runtime-host', url.origin);
```

Comment:
This introduced a regression. It used to be possible to specify a host like `https://example.com/amp-runtime`. This strips away `/amp-runtime`, resulting in incorrect URLs for dynamically loaded extensions.